### PR TITLE
Updated PR from cjharkins

### PR DESCRIPTION
**Pull request by ganeshpatro321**
Changes: 
 - Hampered UI restructured
 - Removed screenshots
 - Settings icon changes

#21

### CODE REVIEW

File: src/components/layouts/Header.js [hunk 1]
```diff
@@ -1,11 +1,35 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import React, { useState } from 'react';
+import Notification from '../notification/Notification';
 
-export default class Header extends React.Component {
-  render() {
-    return (
+export default function Header() {
+  const [showNotification, setShowNotification] = useState(false);
```
Comment (Harkishen-Singh) [inline src/components/layouts/Header.js]: please change this to class-based component

File: src/components/layouts/Header.js [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
+
+  const updateShowNotificationsScreen = () => {
+    setShowNotification(!showNotification);
+  };
+
+  return (
+    <>
```
Comment (Harkishen-Singh) [inline src/components/layouts/Header.js]: please make this `<div>` to avoid readability issues.

Comment (ganeshpatro321) [reply src/components/layouts/Header.js]: Lets make use of fragments.

**Comment by ganeshpatro321** [comment]
Ready to merge?

**Pull request merged**